### PR TITLE
feat (OT267-25): endpoint registro de usuarios

### PR DESCRIPTION
**Pull request by zayrarepositor**
https://alkemy-labs.atlassian.net/browse/OT267-25 - Endpoint registro de usuarios

## Actions:
:bomb: bcryptjs and express-validator installed

:white_check_mark: post request for user creation
:white_check_mark: validations for firstName, lastName, email and password: 
  * firstName: required, length(1-15)  
  * lastName: required, length(1-15) 
  * email: required, unique, sanitization
  * password: length(min 8), uppercase(min 1), number(min 1), passwordConfirmation

:white_check_mark: encrypted password
:white_check_mark: user created as response

## Snips:
<p align='center'>user post</p>

user post and password encrypt

<p align='center'>validations: required</p>

2 -validation first name, lastname

<p align='center'>validations 2:</p>

3 - validation first name, lastname and email

<p align='center'>sanitization: mail</p>

3 - sanitization email

<p align='center'>verification: unique mail</p>

3 - validation mail unique

Good Life! ( ͡~ ͜ʖ ͡°)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-    "express": "~4.16.1",
-    "http-errors": "~1.6.3",
-    "morgan": "~1.9.1",
-    "mysql2": "^2.2.3",
-    "sequelize": "^6.3.5"
-  },
-  "devDependencies": {
-    "sequelize-cli": "^6.2.0"
-  }
+    "name": "ong-server",
```
Comment (FerLeiva) [inline package.json]: No deberias agregar este archivo al pull request

**Review by FerLeiva — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (zayrarepositor) [reply package.json]: done

**Comment by FerLeiva** [comment]
Zayra por favor resolve ese conflicto antes de hacer el marge. Gracias!

**Comment by zayrarepositor** [comment]
Done! @FerLeiva

**Pull request merged**